### ISSUE TITLE
Crash on export backup (any)

### ISSUE THREAD

**Issue by Wjxfi** [ammargitham/WallFlow#17]
```
FATAL EXCEPTION: main
Process: com.ammar.wallflow, PID: 10120
java.lang.IllegalArgumentException: Unknown URI: content://downloads/public_downloads/5279
	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:172)
	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:142)
	at android.content.ContentProviderProxy.query(ContentProviderNative.java:481)
	at android.content.ContentResolver.query(ContentResolver.java:1219)
	at android.content.ContentResolver.query(ContentResolver.java:1151)
	at android.content.ContentResolver.query(ContentResolver.java:1107)
	at kotlin.ResultKt.getDataColumn(SourceFile:1)
	at kotlin.ResultKt.getRealPath(SourceFile:156)
	at coil.util.-Calls.access$BackupDialogContent(SourceFile:757)
	at androidx.compose.animation.CrossfadeKt$Crossfade$5$1.invoke(SourceFile:143)
	at androidx.compose.animation.CrossfadeKt$Crossfade$5$1.invoke(SourceFile:4)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
	at androidx.compose.runtime.ComposerImpl.recomposeToGroupEnd(SourceFile:366)
	at androidx.compose.runtime.ComposerImpl.skipToGroupEnd(SourceFile:29)
	at androidx.room.Room.UnpaddedAlertDialogContent-Pqc3_vk(Unknown Source:381)
	at androidx.compose.material3.AppBarKt$SingleRowTopAppBar$2.invoke(SourceFile:5)
	at androidx.compose.material3.AppBarKt$SingleRowTopAppBar$2.invoke(SourceFile:1)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
	at _COROUTINE._BOUNDARY$$ExternalSyntheticOutline0.m(SourceFile:66)
	at androidx.compose.material3.AndroidAlertDialog_androidKt$AlertDialog$3.invoke(SourceFile:32)
	at androidx.compose.material3.AndroidAlertDialog_androidKt$AlertDialog$3.invoke(SourceFile:2)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(SourceFile:8)
	at androidx.compose.runtime.ComposerImpl.recomposeToGroupEnd(SourceFile:366)
	at androidx.compose.runtime.ComposerImpl.skipCurrentGroup(SourceFile:58)
	at androidx.compose.runtime.ComposerImpl.doCompose(SourceFile:169)
	at androidx.compose.runtime.ComposerImpl.recompose$runtime_release(SourceFile:29)
	at androidx.compose.runtime.CompositionImpl.recompose(SourceFile:21)
	at androidx.compose.runtime.Recomposer.access$performRecompose(SourceFile:85)
	at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2$1.invoke(SourceFile:381)
	at androidx.compose.ui.platform.AndroidUiFrameClock$withFrameNanos$2$callback$1.doFrame(Unknown Source:6)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.doFrame(SourceFile:48)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1140)
	at android.view.Choreographer.doCallbacks(Choreographer.java:946)
	at android.view.Choreographer.doFrame(Choreographer.java:870)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1127)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:210)
	at android.os.Looper.loop(Looper.java:299)
	at android.app.ActivityThread.main(ActivityThread.java:8261)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:556)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1045)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@ebe1e57, androidx.compose.ui.platform.MotionDurationScaleImpl@27b9d44, StandaloneCoroutine{Cancelling}@f24de2d, AndroidUiDispatcher@2327f62]
```

**Comment by ammargitham** [comment]
Fixed in [v1.5.0](https://github.com/ammargitham/WallFlow/releases/tag/v1.5.0)

**Comment by ammargitham** [comment]
@Wjxfi Please test on v1.5.0 and let me know if the crash is fixed.
